src/Version10/Definition/Builder/ConsoleOptionBuilderInterface.php

```diff
@@ -53,7 +57,8 @@ public function withShortcutOnly(): self;
     /**
      * Declare that a value is optional for this option. A default value may be defined also.
      *
-     * If multiple option values are supported, the method may be called multiple times.
+     * You may only declare a value as required or optional. If multiple option values are supported, the method may
```

Comment:
Can you clearify this?
Does this mean, when 2 values are supported, I call it as:
```
$builder
    ->withOptionalValue('file')
    ->withOptionalValue('file')
```
If so, how to allow for an arbitrary amount of values (ijn this case file names)?

Reply:
The support for multiple calls is designed for options like `--init [source_dir=src] [level=3]` where two named values are possible. There is no way to declare an arbitrary amount of values right now, but it's possible to define an option as multiple  with `isArray()`.

Reply:
Improved documentation in 328d6b6.